src/screens/finished.js

```diff
@@ -166,16 +171,26 @@ function Finished() {
           setRadioButtonThatIsActivated(value);
         }}
       />
-      <FlatList
-        style={{ marginTop: 15 }}
-        data={
-          radioButtonThatIsActivated === 'ALL'
-            ? contributors
-            : filteredContributors.sort((res) => res.contributions)
-        }
-        renderItem={contributorItemFromList}
-        keyExtractor={(item) => String(item.id)}
-      />
+      {contributors.length === 0 && (
+        <ActivityIndicator
+          isVisible={contributors.length === 0}
```

Comment:
It’t the same as `isVisible` since we already have a display condition … We have to choose between 

```tsx
{!contributors.length && (
  <ActivityIndicator isVisible size={40} color="#2855a2" />
)}
```

and 

```tsx
<ActivityIndicator
  isVisible={!contributors.length}
  size={40}
  color={'#2855a2'}
/>
```